proto/demo_grpc/Makefile.am

```diff
@@ -8,7 +8,7 @@ AM_CPPFLAGS = \
 -I$(top_builddir)/grpc_out \
 -I$(top_srcdir)/p4info
 
-AM_CXXFLAGS = -Wall -Werror
+AM_CXXFLAGS = -Wall -Werror -Wno-unused-command-line-argument
```

Comment:
Could you add a comment explaining the `-Wno-unused-command-line-argument` addition?

Reply:
linking failed since -lpthread was an unused linker argument. 
clang: error: argument unused during compilation: '-pthread' [-Werror,-Wunused-command-line-argument]
But I looked at this more, seems to be just a problem with xcode/clang. It's just on the demo grpc code, let me know what's preferred

Reply:
I think you should try removing the `AM_CXXFLAGS` line altogether. The configure.ac should make sure the flags are correct already.

Reply:
Just tried that, now getting multiple warnings->error in the pb header:
../grpc_out/p4/v1/p4runtime.grpc.pb.h:210:59: error: unused parameter 'service' [-Werror,-Wunused-parameter]

Reply:
removing AM_CXXFLAGS shows that -Wextra is now on the compile line, which causes the detection of the unused parameter errors